### PR TITLE
Add Lore Section

### LINKED ISSUE #19: Add Lore Section
This society has a lot of lore as evidenced by https://discord.com/channels/504971400394702858/550266698880516106/1157614584182620280 and https://discord.com/channels/504971400394702858/552761101033406475/1157498483088031766 which fade away over time if not documented somewhere. 

Other examples include the 2 bricks that the society has and the efforts that people put in throughout the years.

### PR DESCRIPTION
**Pull request by sebasptsch**
Fixes #19

### CODE REVIEW

**Review by arduano — APPROVED** [review]
Not sure why you need my approval for this, but lgtm

**Comment by arduano** [comment]
Unless you want me to contribute to the lore content? Which I can do

File: docs/lore/brick.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+title: Brick
+description: ProgSoc has 2 bricks. Why? Because we can.
+---
+
+Visit the [Brick](https://brick.progsoc.org) website which is delighfully early 2000s.
```
Comment (sunsetkookaburra) [inline docs/lore/brick.md:6]: perfect

**Review by sunsetkookaburra — APPROVED** [review]
i wonder if stuff about the room would fit in lore, or historical archives (or are they the same?).
definitely should have TFM though

**Pull request merged**